packages/editor/src/components/global-styles-provider/index.js

```diff
@@ -118,7 +118,7 @@ function useGlobalStylesBaseConfig() {
 	return [ !! baseConfig, baseConfig ];
 }
 
-function useGlobalStylesContext() {
+export function useGlobalStylesContext() {
```

Comment:
Should we make this private?

Reply:
oh it is!

Reply:
Yep, it's not exported from the editor package at this stage 👍